sources/packages/backend/libs/test-utils/src/factories/application-offering-change-request.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ApplicationOfferingChangeRequest,
+  ApplicationOfferingChangeRequestStatus,
+  ApplicationStatus,
+  EducationProgramOffering,
+  InstitutionLocation,
+  User,
+} from "@sims/sims-db";
+import { saveFakeApplicationDisbursements } from "./application";
+import { DataSource } from "typeorm";
+import { createFakeEducationProgramOffering } from "./education-program-offering";
+import { createFakeUser } from "./user";
+import * as faker from "faker";
+
+/**
+ * Create and save a fake application offering request change record.
+ * @param dataSource manages the repositories to save the data.
+ * @param relations dependencies:
+ * - `institutionLocation` related location.
+ * @param options additional options:
+ * - `applicationOfferingChangeRequestStatus` application offering change request status.
+ * @returns created and saved application offering request change record.
+ */
+export async function saveFakeApplicationOfferingRequestChange(
+  dataSource: DataSource,
```

Comment:
I believe that the idea would pass the `E2EDataSource` to avoid the `getRepository` inside these methods also.